build/lib/extensions.ts

```diff
@@ -308,7 +308,6 @@ const externalExtensions = [
 	'dacpac',
 	'import',
 	'kusto',
-	'liveshare',
```

Comment:
the js file generated by this should also be included. run `yarn compile` in the /build folder, and the extensions.js will be updated.